TruthTables/Tests.qs

```diff
@@ -20,46 +20,53 @@ namespace Quantum.Kata.TruthTables {
         Fact(actual::bits == expected::bits, $"Truth table for {name} is not correct");
     }
 
+    @Test("Microsoft.Quantum.Katas.CounterSimulator")
```

Comment:
TruthTables kata doesn't work when executed locally - it gives me "Invalid test name: T1_ProjectiveTruthTables_Test" which typically indicates missing "%workspace reload" command. 

Alternatively, I think this kata should work with `QuantumSimulator`, it doesn't look like it's using any of the `CounterSimulator` features - can you try that?

Reply:
Let's handle switching from `CounterSimulator` to `QuantumSimulator` in a separate PR - it's going to be easier to keep this a purely syntax change and to review and troubleshoot meaningful changes with smaller scope